src/components/dialogs/resubmit.vue

```diff
@@ -204,11 +223,12 @@ export default class ResubmitDialog extends Mixins(
       }
     }
 
-    const success = await this.createPayment({
+    // if resubmit succeeded then create the payment
+    success = success && await this.createPayment({
       action: PaymentAction.RESUBMIT,
       nrId: this.getNrId,
       filingType: FilingTypes.NM620,
```

Comment:
Outstanding question: should this be type NM620 or some other type for resubmit?